operators/src/processing/point_in_polygon.rs

```diff
@@ -187,55 +187,78 @@ impl VectorQueryProcessor for PointInPolygonFilterProcessor {
 ///
 /// The algorithm is taken from <http://alienryderflex.com/polygon/>
 ///
-struct PointInPolygonTester<'p> {
-    polygons: &'p MultiPolygonCollection,
+pub struct PointInPolygonTester {
+    polygons: MultiPolygonCollection,
     constants: Vec<f64>,
     multiples: Vec<f64>,
 }
 
-impl<'p> PointInPolygonTester<'p> {
-    pub fn new(polygons: &'p MultiPolygonCollection) -> Self {
+impl PointInPolygonTester {
+    pub fn new(polygons: MultiPolygonCollection) -> Self {
         let number_of_coordinates = polygons.coordinates().len();
 
-        let mut tester = Self {
+        let mut constants = vec![0.; number_of_coordinates];
+        let mut multiples = vec![0.; number_of_coordinates];
+
+        Self::precalculate_polygons(&polygons, &mut constants, &mut multiples);
```

Comment:
i would move constants and multiple vec creation into the method

Reply:
9f23d098a108d4f47eb087a79770ced81eabfe5c